arbnode/blockmetadata.go

```diff
@@ -28,18 +28,17 @@ type BlockMetadataFetcherConfig struct {
 }
 
 var DefaultBlockMetadataFetcherConfig = BlockMetadataFetcherConfig{
-	Enable:         false,
```

Comment:
I suspect this is actually breaking some tests that were not expecting to need to provide a source for BlockMetadata. See the failures in CI.

Reply:
1. Should only be enabled for timeboost chains
2. Need to auto-populate source URL for arb1/nova

Reply:
enabled on a per-chain basis